pwnedapi/tests/scanner_test.py

```diff
@@ -25,7 +27,29 @@ def emptyfile():
     return NamedTemporaryFile().name
 
 
+@responses.activate
 def test_export_as(tempfile):
+    dog = Password("dog")
```

Comment:
Could these variables and calls to `responses.add` be done with list iteration to make it more compact?